python/pyspark/sql/functions.py

```diff
@@ -582,6 +582,43 @@ def nanvl(col1, col2):
     return Column(sc._jvm.functions.nanvl(_to_java_column(col1), _to_java_column(col2)))
 
 
+@since(3.1)
+def percentile_approx(col, percentage, accuracy=10000):
```

Comment:
`accuracy` here too. Let's make it able to take both int and a column instance. Should be able to do it easily by calling `lit` on it.